src/engine/controls/keycontrol.cpp

```diff
@@ -94,6 +94,10 @@ KeyControl::KeyControl(const QString& group,
     m_pKeylock = new ControlProxy(group, "keylock", this);
     m_pKeylock->connectValueChanged(this, &KeyControl::slotRateChanged,
             Qt::DirectConnection);
+    // Internal m_pitchRateInfo.keylock is updated only in updateRate() when any
+    // rate, keylock or vc controls are touched. Though, it may be requested by
+    // EngineBuffer, e.g when nudging a track. Thus, update it now.
+    m_pitchRateInfo.keylock = m_pKeylock->get();
```

Comment:
oo I like the inline clazy notes, very fancy

Reply:
```suggestion
    m_pitchRateInfo.keylock = m_pKeylock->toBool();
```

Reply:
clazy?
crazy & lazy

Reply:
oh you mean the CI
already fixed